rct229/ruleset_functions/compare_standard_val.py

```diff
@@ -0,0 +1,31 @@
+from rct229.utils.assertions import assert_
+from rct229.utils.std_comparisons import std_equal
+
+
+def compare_standard_val(val, std_val, operator=None, ahj_ra_compare: bool = False
+                ) -> bool:
+    """Determines whether the model value and standard value are equal or match to user specified relationship
+    if stringent code check applied.
+
+    Parameters
+    ----------
+    val: Quantity
+        value extracted from model
+    std_val : Quantity
+        standard value from code
+    operator: operator
+        defined user relationship, it should be either operator.lt,operator.le, operator.gt, operator.ge
+    ahj_ra_compare: Boolean
+        flag True: stringent code check, False: regular equality check
+
+    Returns
+    -------
+    bool
+        True if the comparison matches to the defined relationship or equal.
+    """
+    if ahj_ra_compare:
+        assert_(operator, "Need to provide operator (operator.lt, operator.gt, operator.le, operator.ge) for "
+                          "stringent code comparison")
+        return std_equal(std_val, val) or operator(val, std_val)
```

Comment:
@weilixu Good morning! I thought we were also going to include the global tolerance in the function logic as well. Maybe I am missing it. 

Also, I have created RDSs that assume a different order of input parameters but the meaning of each is clearly explained in the RDS. Is this OK or do I need to go back and update it.